README.md

```diff
@@ -13,47 +13,61 @@ The [Okta developer site][doc] serves Okta's API documentation and guides, inclu
 
 ### Requirements
- - Node: 8+
- - Yarn: 1.7+
+ - [Node](https://nodejs.org/en/download/): 8+
+ - [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable): 1.7+
 
-### Installing the site.
- - Clone this repository (or fork if you aren't a core contributor):
+Before getting started, open a terminal window and make sure these commands work:
+
+```sh
+node --version
+
+yarn --version
+```
+
+### Local setup
+ 1. Fork this repository (if you're an Okta employee, skip this step)
+ 2. Clone
 
  ```sh
  git clone git@github.com:okta/okta-developer-docs.git
  ```
 
-- Install the dependencie with `yarn`:
+3. Install the dependencies with `yarn`:
 ```sh
+cd okta-developer-docs
+
 yarn install
 ```
 
-This will install all the modules you need for the site to run on your machine
+This will install everything you need to build the documentation on your machine.
 
-### Viewing the Site Locally
-Once you have the site cloned to your machine and installed, you will have to run the development environment to view the site.
+### Building the site
 
- - Open Terminal and change directories into your cloned repo
- - Issue the `yarn dev` command
- - Once the command is done running, you can visit http://localhost:8080/documentation/ in your web browser to view the site.
- > Note: if you try to visit the root, you will get a 404 page.  You must visit a sub-path.
+With the above steps completed, you can start a local server by running this command inside the cloned directory:
 
-## Adding and Updating Content
-As an example, lets say you want to edit the [Okta Angular Sign-in Widget](https://developer.okta.com/code/angular/okta_angular_sign-in_widget/) code page. The URL of this page is `/code/angular/okta_angular_sign-in_widget/`.
-To edit this page, you would navigate to `/packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md` to edit the content of this page.
+```sh
+yarn dev
+```
 
-The directory structure from the `/packages/@okta/vuepress-site` folder relates directly to the URL you want to show. The `index.md` file that lives in the last folder in the structure will be rendered.
-If you name the file anything other than `index.md`, it will be required that you also include `.html` to the url when you go to view the page in the browser
+This starts a preview server on your machine, and watches all files for changes. Open http://localhost:8080/documentation/ to view the documentation.
 
-Part of the power that the VuePress development server gives us is live reloading of the content. When you make any changes to the markdown of the file, you will see the changes on the browser within seconds.
-However, if you make any chnages to the [front matter](https://github.com/vuejs/vuepress/blob/master/packages/docs/docs/guide/frontmatter.md) of the page, you may have to restart the development server.
+ > Note: if you try to visit the root, you will get a 404 page.  You must visit a path like `/documentation`.
```

Comment:
It is not any path.  it is only paths that are folders within the `packages/@okta/vuepress-site` folder